omega/omega-connector/omega-connector-grpc/src/main/java/org/apache/servicecomb/saga/omega/connector/grpc/LoadBalancedClusterMessageSender.java

```diff
@@ -20,24 +20,29 @@
 import static java.util.Collections.emptyList;
 import static java.util.concurrent.TimeUnit.MILLISECONDS;
 
+import com.google.common.base.Supplier;
```

Comment:
You may need to import the google code formate [here](http://servicecomb.incubator.apache.org/developers/setup-develop-environment/).